innsender/src/main/kotlin/no/nav/soknad/innsending/service/FilService.kt

```diff
@@ -94,16 +96,20 @@ class FilService(
 			}
 		}
 
-		if (soknadDto.vedleggsListe.none { it.id == filDto.vedleggsid })
+		val vedleggdto = soknadDto.vedleggsListe.filter { it.id == filDto.vedleggsid }.getOrNull(0)
+		if (vedleggdto == null)
 			throw ResourceNotFoundException("Vedlegg ${filDto.vedleggsid} til søknad ${soknadDto.innsendingsId} eksisterer ikke")
 
-		logger.debug("${soknadDto.innsendingsId!!}: Skal lagre fil med størrelse ${filDto.data!!.size} på vedlegg ${filDto.vedleggsid}")
+		val start = System.currentTimeMillis()
+		logger.info("${soknadDto.innsendingsId!!}: Skal lagre fil med størrelse ${filDto.data!!.size} på vedlegg ${filDto.vedleggsid}")
```

Comment:
Denne vil feile dersom filDto.data er null. Vet ikke om det faktisk kan inntreffe, men `mapTilFilDb(filDto)` tar iallefall høyde for det, så kanskje tryggest å gjøre det her også?

Reply:
Endret til ${filDto.data?.size ?: 0}